src/Collection/Collection.model.ts

```diff
@@ -91,16 +110,28 @@ export class Collection extends Model<CollectionAttributes> {
         ? [
             CurationStatusFilter.PENDING,
             CurationStatusFilter.APPROVED,
+            CurationStatusFilter.REJECTED,
           ].includes(status)
-          ? SQL`collection_curations.status = ${status} AND collections.contract_address = ANY(${remoteIds})`
-          : status === CurationStatusFilter.REJECTED // Rejected not a single curation in approved
-          ? SQL`collection_curations.status = ${status} AND collections.contract_address = ANY(${remoteIds})`
+          ? SQL`(collection_curations.status = ${status} AND (collections.contract_address = ANY(${remoteIds}) OR ${raw(
```

Comment:
I really like what you did to re-use the SQL statements above, I was wondering if we can do the same for 
`collections.contract_address = ANY(${remoteIds}`
maybe we can have one variable like the following and re-use it among the queries
```
const isInRemoteIds = SQL`collections.contract_address = ANY(${remoteIds}`
```